app/operations/transfers/to_service.rb

```diff
@@ -113,7 +113,8 @@ def update_transfer(response)
         status = xml.xpath('//tns:ResponseDescriptionText', 'tns' => 'http://hix.cms.gov/0.1/hix-core')
         status_text = status.any? ? status.last.text : "N/A"
         payload = status_text == "Success" ? "" : @transfer.outbound_payload
-        @transfer.update!(response_payload: response_json, callback_status: status_text, outbound_payload: payload, failure: nil)
+        failure = response[:status] == 200 ? nil : "Response failure not a #{response[:status]}"
```

Comment:
I don't quite understand the error message.  Is there a way we can make it more clear?  It sounds like we are saying the response is not the status that it actually is.